<?xml version="1.0" encoding="UTF-8"?>
<testsuite name="com.yahoo.test.SelNG.YUI.tests.DragAndDropTestShim"
           tests="1"
           failures="1"
           errors="0"
           skipped="0"
           time="56.632">
  <properties />
      <testcase name="DragAndDropShim" time="56.632">
            <failure type="java.lang.AssertionError" 
                  message="null"
              >
<![CDATA[java.lang.AssertionError: null
  org.testng.Assert.fail(Assert.java:86)
  org.testng.Assert.fail(Assert.java:93)
  com.yahoo.test.SelNG.YUI.tests.DragAndDropTestShim.DragAndDropShim(DragAndDropTestShim.java:23)
  sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
  sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:39)
  sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:25)
  java.lang.reflect.Method.invoke(Method.java:597)
  org.testng.internal.MethodHelper.invokeMethod(MethodHelper.java:607)
  org.testng.internal.Invoker.invokeMethod(Invoker.java:517)
  org.testng.internal.Invoker.invokeTestMethod(Invoker.java:669)
  org.testng.internal.Invoker.invokeTestMethods(Invoker.java:956)
  org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
  org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:110)
  org.testng.internal.thread.ThreadUtil$CountDownLatchedRunnable.run(ThreadUtil.java:132)
  java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:886)
  java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:908)
  java.lang.Thread.run(Thread.java:637)
]]>
      </failure>
          </testcase>
      </testsuite>
